site/_posts/2017-01-16-destroy-the-monolith.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Last Week in ManageIQ: Destroy the Monolith"
+author: durandom
+date: 2017-01-16 09:24:51 +01:00
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+[Moin](https://en.wikipedia.org/wiki/Moin), [durandom][https://github.com/durandom/] from the providers team here, 
+guiding you through the vast construction site of [ManageIQ](http://manageiq.org/).
+
+Last week we had [206 PRs][PRs merged last week] merged in manageiq core.
+Not counting all the [related repos](https://github.com/ManageIQ?utf8=%E2%9C%93&q=&type=source), 
+which we continue to carve out of our beloved Majestic Monolith. 
+
+You have to destroy to create.
+
+## Featured
+It was a small step for [Zita](https://github.com/ZitaNemeckova) to 
+[rename](https://github.com/ManageIQ/manageiq-ui-classic/pull/150) 'Automate' to 'Automation' in the
+UI, but it might be a giant leap for ManageIQ.
+
+Stay true to the game: [Tim](https://github.com/imtayadeway) and [Sam](https://github.com/mansam) thought that should 
+apply to tests as well and pushed our specs closer to reality. Quoting them: 
```

Comment:
ah, yes. Now I understand. Thanks for the clarification @JPrause - nice suggestion.